.travis/quay-push.sh

```diff
@@ -18,3 +18,9 @@ IMAGE_TAG=${QUAY_USER_OR_ORG_NAME:-latest}
 # Reference: https://adriankoshka.github.io/blog/posts/travis-and-quay/
 echo "$QUAY_BOT_PASSWORD" | docker login -u "$QUAY_BOT_USERNAME" --password-stdin quay.io
 docker push "quay.io/$QUAY_PROJECT_NAME/$QUAY_REPO_NAME:$IMAGE_TAG"
+
+cd $TRAVIS_BUILD_DIR/../pulpcore/containers/
+pip install ansible
+ansible-playbook -v build.yaml
+echo "$QUAY_BOT_PASSWORD" | docker login -u "$QUAY_BOT_USERNAME" --password-stdin quay.io
+docker push "quay.io/$QUAY_PROJECT_NAME/pulp:$IMAGE_TAG"
```

Comment:
So this builds everything:
```
pulpcore:stable
pulpcore:latest
pulp:latest
pulp:stable
```

But it only pushes and tests `pulp:latest`.

It also doesn't keep any versions or latest by giving them timestamps tags, which is useful, but not essential. And would require pruning old versions.

We had an epic for this back when we assumed that we would use pulp-operator in the plugin-template.

https://pulp.plan.io/issues/5393

I think we should go back to design for this PR. Even if we end up making 0 code changes to it, and focus on implementing the rest of the rewritten epic later. Our Pulp Installers meeting is tomorrow morning anyway, we can do it then.

Reply:
I was not aware of this epic, and I agree we should go back to it. The only action we should take for now is to update pulp image (manually), 3.0.1 is too old.
Maybe close this PR?

Reply:
I just realized we forgot to discuss this.

I think you should copy the defaults.yaml to vars.yaml and modify it with yq, or just provide a vars.yaml in this repo. It will only build pulpcore:master and pulp:master, and you will only push those.

The rest of the work should be addressed as part of the Epic.

Reply:
I see the latest commit. Please add and push pulpcore:master as well. Its final layer should be identical to an intermediate layer in the pulp:master image, so the performance cost should be negligible.